src/creds.jl

```diff
@@ -22,9 +22,8 @@ struct AccessToken
 end
 
 function isexpired(access_token::AccessToken)::Bool
-    expires_in = access_token.expires_in * 1000  # millis
-    expires_on = access_token.created_on + expires_in
-    return expires_on > now()
+    expires_on = access_token.created_on + Second(access_token.expires_in)
+    return expires_on < now()
 end
```

Comment:
Sorry I'm not very familiar with this code, but would it be a good idea to add a couple of seconds of buffer so you don't get into problems that it is expiring right when being submitted?

Reply:
good idea! 

The python SDK has some similar logic to this; can you check to see what we do there, and if we don't do this, can you file an issue to fix it there too? (and/or in all the SDKs)

Reply:
Should be addressed as part of this PR: https://github.com/RelationalAI/rai-sdk-julia/pull/33
Will try to cover the same requirement for other SDKs :pray: